src/components/bcros/filing/addStaffFiling/modal/Form.vue

```diff
@@ -7,6 +7,7 @@ const t = useNuxtApp().$i18n.t
 const filings = useBcrosFilings()
 const business = useBcrosBusiness()
 const { currentBusiness, currentBusinessName } = storeToRefs(business)
+const { enableDocumentRecords } = storeToRefs(useBcrosDocuments())
```

Comment:
Is there a missing file from this commit? Where is the FF code?

Reply:
I believe Brandon is leverage the computed property(that includes the FF) put in previously for the Document Record Link.

Reply:
I looked deeper and confirmed this. You are right.

This PR's description had me expecting more, lol.

> Implemented a feature flag strategy.